File: BTT4/BTT4 (BTT3 UPDATE)/BTT3/DisplayMessUserSentGUI.cpp
#include "DisplayMessUserSentGUI.h"
#include "Controller.h"

void DisplayMessUserSentGUI::RequestDisplay(string userID)
{
	Controller Controller_GUI;
//	cout << "input messID: " << endl;
	//string messID;
	//cin >> messID;
	Controller_GUI.RequestDisplay(userID);
}

DisplayMessUserSentGUI::DisplayMessUserSentGUI()
{
}


DisplayMessUserSentGUI::~DisplayMessUserSentGUI()
{
}

File: BTT4_PY1/BTT4_PY1/DisplayMessUserSentGUI.py
from Controller import Controller
class DisplayMessUserSentGUI(object):
	def RequestDisplayMessUserSentGUI(userID):
		controller=Controller
		#controller.RequestDisplayMessUserSentGUI(userID)
		#print(controller.RequestDisplayMessUserSentGUI(userID))
		for row in controller.RequestDisplayMessUserSent(userID):
			print(row[0])


